include/drawtypes/iconset.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <map>
+#include <algorithm>
```

Comment:
Can (and should) be moved to the cpp file

Reply:
moved to iconset.cpp